include/bluetooth/gatt.h

```diff
@@ -625,7 +625,13 @@ ssize_t bt_gatt_attr_read_chrc(struct bt_conn *conn,
 #if IS_ENABLED(CONFIG_BT_SETTINGS_CCC_LAZY_LOADING)
 	#define BT_GATT_CCC_MAX (CONFIG_BT_MAX_CONN)
 #else
-	#define BT_GATT_CCC_MAX (CONFIG_BT_MAX_PAIRED + CONFIG_BT_MAX_CONN)
+	#if defined(CONFIG_BT_MAX_PAIRED) && defined(CONFIG_BT_MAX_CONN)
+		#define BT_GATT_CCC_MAX (CONFIG_BT_MAX_PAIRED + CONFIG_BT_MAX_CONN)
+	#elif defined(CONFIG_BT_MAX_CONN)
+		#define BT_GATT_CCC_MAX (CONFIG_BT_MAX_CONN)
+	#else
+		#define BT_GATT_CCC_MAX 0
+	#endif
```

Comment:
```
	#if defined(CONFIG_BT_CONN)
		#define BT_GATT_CCC_MAX (CONFIG_BT_MAX_PAIRED + CONFIG_BT_MAX_CONN)
	#else
		#define BT_GATT_CCC_MAX 0
	#endif
```

To simplify it a bit, as both `CONFIG_BT_MAX_PAIRED` and `CONFIG_BT_MAX_CONN` are defined if `CONFIG_BT_CONN` is defined.

Reply:
Ok, changed